### ISSUE TITLE
Cactus [toil] - Not enough memory

### ISSUE THREAD

**Issue by francicco** [DataBiosphere/toil#3191]
Hi,

I'm running [Cactus](https://github.com/ComparativeGenomicsToolkit/cactus) on my cluster, which has a total of 500Gb of memory. Apparently isn't enough because toil keep returning me `Not enough memory!` Error type of message.

@MarkFilus is saying that I should contact you guys for the [issue](https://github.com/ComparativeGenomicsToolkit/cactus/issues/310#issuecomment-687635144)

Is there anything I could do to fix the problem?

Thanks a lot
F

**Comment by diekhans** [comment]
I don't think this case actually needs more memory, but has gone down some path of requesting more until it runs out.
